partner_delivery_schedule/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_delivery_schedule_user,access_delivery_schedule_user,model_delivery_schedule,base.group_partner_manager,1,0,0,0
```

Comment:
I think you should add a general rule read only for employees, as they access contacts although they don't have that permission. Salesman should be also allowed to change delivery times (more than stock users). See delivery module for copying the permission scheme with delivery carrier.